go.mod

```diff
@@ -1,16 +1,12 @@
 module github.com/CrowdStrike/foundry-fn-go
 
-go 1.20
+go 1.21
 
-require (
-	github.com/crowdstrike/gofalcon v0.3.1
-	github.com/stretchr/testify v1.8.2
-)
+require github.com/crowdstrike/gofalcon v0.3.1
```

Comment:
hey @jsteenb2 not sure what all is in scope for this, but thoughts on bumping the version of gofalcon that is included? There have been some bugfixes like https://github.com/CrowdStrike/gofalcon/pull/368 & `v0.4.0` introduced many new endpoints and operations from the api spec. There will be a new release this week to include all the changes over the past month which will include some new collections.

Reply:
sure thing, we can get that version upgraded here 👍

Reply:
excellent, I'll wait to bring that in before I merge 👍

Reply:
Delayed to implement another collection that he wants & he found a few things wrong with the current spec. I'll comment back when this is done.

Reply:
we can always add a patch version to `v0.4.1`, I won't block on this update for now. I have some work to do, but we're more than capable of rolling a new version with small updates anytime 👍